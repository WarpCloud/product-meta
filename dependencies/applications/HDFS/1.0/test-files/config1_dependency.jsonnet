{
  application_type: "HDFS",
  application_name: "hdfs1",
  application_version: "1.0",
  user_config: {},
  app_config: {
    "hdfs_name_cpu_limit": 20,
  },
  development_mode: true,
}
