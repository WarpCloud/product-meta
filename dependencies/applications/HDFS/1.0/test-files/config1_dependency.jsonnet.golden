{
   "Customized_Namespace": "default",
   "Develop": true,
   "TCU": {
      "hdfs": {
         "c": 0.875,
         "m": 0.69999999999999996,
         "s": 0.875
      },
      "zookeeper": {
         "c": 0.125,
         "m": 0.10000000000000001,
         "s": 0.125
      }
   },
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "hdfs1",
            "application_type": "HDFS"
         },
         "configs": {
            "zk_cpu_limit": 0.20000000000000001,
            "zk_cpu_request": 0.20000000000000001,
            "zk_memory_limit": 1,
            "zk_memory_request": 1,
            "zk_storage_config": {
               "accessModes": [
                  "ReadWriteOnce"
               ],
               "limits": {
                  "blkio.throttle.read_iops_device": 100,
                  "blkio.throttle.write_iops_device": 100
               },
               "size": "10Gi",
               "storageClass": "silver"
            }
         },
         "instance_settings": {
            "hdfs_name_cpu_limit": 20
         },
         "moduleName": "zookeeper",
         "name": "hdfs1-zookeeper",
         "version": "1.0"
      },
      {
         "annotations": {
            "application_name": "hdfs1",
            "application_type": "HDFS"
         },
         "configs": {
            "hdfs_data_cpu_limit": 0.20000000000000001,
            "hdfs_data_cpu_request": 0.20000000000000001,
            "hdfs_data_memory_limit": 1,
            "hdfs_data_memory_request": 1,
            "hdfs_data_storage": {
               "accessModes": [
                  "ReadWriteOnce"
               ],
               "limits": {
                  "blkio.throttle.read_iops_device": 100,
                  "blkio.throttle.write_iops_device": 100
               },
               "size": "10Gi",
               "storageClass": "silver"
            },
            "hdfs_data_tmp_storage": {
               "accessMode": "ReadWriteOnce",
               "limits": {
                  "blkio.throttle.read_iops_device": 100,
                  "blkio.throttle.write_iops_device": 100
               },
               "size": "10Gi",
               "storageClass": "silver"
            },
            "hdfs_httpfs_cpu_limit": 0.20000000000000001,
            "hdfs_httpfs_cpu_request": 0.20000000000000001,
            "hdfs_httpfs_memory_limit": 1,
            "hdfs_httpfs_memory_request": 1,
            "hdfs_journal_cpu_limit": 0.20000000000000001,
            "hdfs_journal_cpu_request": 0.20000000000000001,
            "hdfs_journal_memory_limit": 1,
            "hdfs_journal_memory_request": 1,
            "hdfs_name_cpu_limit": 0.40000000000000002,
            "hdfs_name_cpu_request": 0.40000000000000002,
            "hdfs_name_data_storage": {
               "accessModes": [
                  "ReadWriteOnce"
               ],
               "limits": {
                  "blkio.throttle.read_iops_device": 100,
                  "blkio.throttle.write_iops_device": 100
               },
               "size": "10Gi",
               "storageClass": "silver"
            },
            "hdfs_name_memory_limit": 2,
            "hdfs_name_memory_request": 2,
            "hdfs_name_tmp_storage": {
               "accessMode": "ReadWriteOnce",
               "limits": {
                  "blkio.throttle.read_iops_device": 100,
                  "blkio.throttle.write_iops_device": 100
               },
               "size": "10Gi",
               "storageClass": "silver"
            },
            "hdfs_zkfc_cpu_limit": 0.40000000000000002,
            "hdfs_zkfc_cpu_request": 0.40000000000000002,
            "hdfs_zkfc_memory_limit": 2,
            "hdfs_zkfc_memory_request": 2
         },
         "dependencies": [
            {
               "moduleName": "zookeeper",
               "name": "hdfs1-zookeeper"
            }
         ],
         "instance_settings": {
            "hdfs_name_cpu_limit": 20
         },
         "moduleName": "hdfs",
         "name": "hdfs1-hdfs",
         "version": "1.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
